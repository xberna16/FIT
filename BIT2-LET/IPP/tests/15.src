.IPPcode18
DEFVAR GF@A
DEFVAR GF@B
DEFVAR GF@Vysledek
MOVE GF@A bool@true
MOVE GF@B bool@false
WRITE string@A=true_B=false
AND GF@Vysledek GF@A GF@B
WRITE string@AND
WRITE GF@Vysledek
OR GF@Vysledek GF@A GF@B
WRITE string@OR
WRITE GF@Vysledek
NOT GF@Vysledek GF@A
WRITE string@NOT_A
WRITE GF@Vysledek
