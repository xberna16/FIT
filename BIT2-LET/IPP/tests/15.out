A=true_B=false
AND
false
OR
true
NOT_A
false
